napari/utils/transforms/transforms.py

```diff
@@ -761,7 +766,11 @@ def linear_matrix(self, linear_matrix):
             )
         )
 
-    @Affine.affine_matrix.setter
+    @property
+    def affine_matrix(self):
+        return super().affine_matrix
+
+    @affine_matrix.setter
```

Comment:
this is caused by inheritance?

Reply:
Having the property explicitly here is because of https://github.com/python/mypy/issues/1465